### DISCUSSION TITLE
Can bulk population that's not included in the comparison groups be used as Batch control?

### DISCUSSION THREAD

**Discussion by tammyyyx** [ImmuneDynamics/Spectre#41] [! News]
Hi Thomas,

Does the batch control sample have to be one of the groups that are being analyzed such as the Mock and Virus groups in the example workflow? Or can it also be a totally different group such as CD4 that is not included in the two groups getting analyzed and is only used for batch correction? If bulk CD4 can be used as batch control how can I only plot the groups of interested (tetramer positive groups in my case) in the umaps and heatmaps and also only analyze the tetramer groups in clustering and statistics?

I'm trying to compare two tetramer-positive groups using Spectre, but due to the very small sample sizes I wanted to try using the bulk CD4 of the same donor as the batch control reference instead of the donor's tetramer-positive cells. However, I noticed that this would cause the difference between the two tetramer groups in the initial plots (umaps) to get masked by the CD4 cells since the CD4 population is so much larger and dominates the plots. The first figure attached is when I only used tetramer cells as the batch control and the second figure is when I used CD4 from the same donor as batch control. I suspect the same thing would occur for the following clustering and statistical analysis. 
![Groups](https://user-images.githubusercontent.com/70506599/114061920-efaa5080-9864-11eb-81ed-7d088ca066a9.png)
![Groups](https://user-images.githubusercontent.com/70506599/114061956-f9cc4f00-9864-11eb-8866-ae216a891021.png)

Thanks!

Regards,

Tammy

**Comment by tomashhurst** [comment]
Hi Tammy, I'll post a more detailed reply later on, but in brief -- you can perform the alignment itself on any population that is a _parent_ of your tetramer positive group. For example, you could use _all T cells_ as for alignment, and once alignment is done, extract just the tetramer pos cell for analysis. This adds some stability as the alignment algorithm gets to see a wide distribution of different markers. Sometimes if trying to do alignment on a small, very specific subset of cells, there isn't enough marker expression for it to perform accurately.

**Comment by tammyyyx** [comment]
Thanks Thomas for the quick explanation! Good to hear about the flexibility. Looking forward to the detailed reply on technicals!
